variables.tf

```diff
@@ -75,7 +75,7 @@ variable "default_node_groups" {
           spot                  = optional(bool, false)
           min_per_az            = optional(number, 0)
           max_per_az            = optional(number, 10)
-          desired_per_az        = optional(number, 1)
+          desired_per_az        = optional(number, 0)
```

Comment:
this is a bit onerous given we need at least 1 compute node. Now there is an install timing issue that we may spin one up and then have it spin down because min = 0 and it may take 10+ minutes before we install anything on the node.
I guess there is a difference between having 3 nodes (1 per AZ) spin up when we only use 1 of them.